### PR TITLE
Update Beat launcher to pass beater config section to New

### PR DESCRIPTION
**Pull request by urso**

### CODE REVIEW

File: generate/beat/{{cookiecutter.beat}}/beater/{{cookiecutter.beat}}.go [hunk 1]
```diff
@@ -36,7 +36,7 @@ func (bt *{{cookiecutter.beat|capitalize}}) Run(b *beat.Beat) error {
 	logp.Info("{{cookiecutter.beat}} is running! Hit CTRL-C to stop it.")
 
 	bt.client = b.Publisher.Connect()
-	ticker := time.NewTicker(bt.config.{{cookiecutter.beat|capitalize}}.Period)
+	ticker := time.NewTicker(bt.config.Period)
```
Comment (ruflin) [inline generate/beat/{{cookiecutter.beat}}/beater/{{cookiecutter.beat}}.go]: One less :-D

File: generate/beat/{{cookiecutter.beat}}/config/config.go [hunk 2]
```diff
@@ -6,15 +6,9 @@ package config
...
-
-type {{cookiecutter.beat|capitalize}}Config struct {
 	Period time.Duration `config:"period"`
 }
 
 var DefaultConfig = Config{
-	{{cookiecutter.beat|capitalize}}: {{cookiecutter.beat|capitalize}}Config{
-		Period: 1 * time.Second,
-	},
+	Period: 1 * time.Second,
```
Comment (ruflin) [inline generate/beat/{{cookiecutter.beat}}/config/config.go]: -3 :-D

**Comment by ruflin** [comment]
LGTM

**Comment by ruflin** [comment]
Winlogbeat still seems to have some complaints here.

**Comment by urso** [comment]
yeah, winlogbeat checks toplevel configs. Unfortunatelly winlogbeat tests for `filters` top-level section, which has been renamed to `prospectors`.

**Comment by ruflin** [comment]
jenkins, retest it

**Pull request merged**